### ISSUE TITLE
[P3][Sourcing][Fdp-combobox] : error state of combobox is not announced by screenreader for a field that has [required]= true

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#13189]
is this a bug, enhancement, or feature request?
Bug

Describe your proposal.
When a fdp-combobox has property [required]= true , and user dont provide or select any value, the screenreader has to read the error state saying the field is required field.
Currently nothing is read by screenreader and only visually red color border appears.

https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/combobox#reactive-form

Can you handle that on the application side
No

Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
ng: 15
core/platform: 0.43.51

If this is a bug, please provide steps for reproducing it; the exact components you are using;
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/combobox#reactive-form

Please provide relevant source code (if applicable).
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/combobox#reactive-form

Please provide stackblitz example(s).

In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
NA
Reported by HMRC customer 740AT.17.11

Did you check the documentation and the API?
Did you search for similar issues?
Is there anything else we should know?
IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone.

**Comment by mikerodonnell89** [comment]
The required state is read by the screenreader:

![Image](https://github.com/user-attachments/assets/d042efb6-fbb2-4754-bede-6ab289221a48)

Are you referring to the "Field is required" text? That would be handled on the application side, as that is provided with this code from the documentation:
```
        @if (errors.required) {
            <span>Field is required</span>
        }
```

**Comment by sagar7259** [comment]
Hello,

On accessing the combo box and when the focus is within the input field, the field is not read as "required" for the users.

Regards,
Sagar

**Comment by I543348** [comment]
> Hello,
> 
> On accessing the combo box and when the focus is within the input field, the field is not read as "required" for the users.
> 
> Regards, Sagar

Hi @mikerodonnell89 ,
From application side we are passing [required]= true but looks like its not reflecting on input element inside combobox.
Hence the issue still exists.
Can you pls check and reopen the issue and help us?
Thanks.

**Comment by mikerodonnell89** [comment]
hi @I543348 What screenreader are you using and also could I see some example code? I am seeing "required" being read

<img width="1534" alt="Image" src="https://github.com/user-attachments/assets/dfbfbd94-1f84-4423-9a45-a42cefadd196" />

**Comment by I543348** [comment]
<img width="750" alt="Image" src="https://github.com/user-attachments/assets/e99df588-76d0-4734-9c20-91d30667a728" />
Hi @mikerodonnell89 ,
pls check this combobox sample code in screenshot , even though we are passing [required]="true" from fdp-combobox , we are not able to see the same on input element, pls note this is not a form field.
Could you pls check and help?

Thanks

**Comment by I543348** [comment]
Hi @mikerodonnell89 , can you please update on this issue?

**Comment by mikerodonnell89** [comment]
> pls check this combobox sample code in screenshot , even though we are passing [required]="true" from fdp-combobox , we are not able to see the same on input element, pls note this is not a form field. Could you pls check and help?
> Thanks

@I543348 I see, in your first post, you had linked the reactive form example, so I was addressing that in my past comments and pull request. I do suppose we can add support a required input on the fdp-combobox itself outside of forms as well. We can reopen this issue to address the problem for non-form comboboxes. 

One thing we might want to do is make `required` from the `BaseInput` abstract class an input - then any component which extends BaseInput will need to handle this. Thoughts @droshev @InnaAtanasova @MariaIDineva ?

**Comment by mikerodonnell89** [comment]
After discussing with my team, we've decided to close this issue as the currently suggested path for requiring inputs in the platform lib is to use them as part of a reactive form. Adding required functionality to the combobox outside of a form will be considered a new feature request
